### DISCUSSION TITLE
Rank up in different languages

### DISCUSSION THREAD

**Discussion by dan-brito** [codewars/codewars.com#2940] [Q&A]
I have overall kyu 5, but my SQL kyu it's 7; but I still get recommendations in SQL language of kyus 5 & 4 (rank up). How do I fix this behavior?

**Comment by Blind4Basics** [comment]
You mostly cannot make the suggestions that precise. On the other hand, you could change the setting to "fundamentals" instead of "rank up", to get easier tasks as suggestions. But that will apply to all your languages as well.
